geode-core/src/main/java/org/apache/geode/management/internal/cli/result/model/SectionResultModel.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.apache.geode.management.internal.cli.result.model;
+
+import java.util.Collection;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+public class SectionResultModel extends AbstractResultModel {
```

Comment:
It seems this is your current model:
InfoResultModel  contains List<String>
TabularResultModel contains Map<String, List<String>>
SectionResultModel contains Map<String, String> and/or TabularResultModel
CompositeResultModel contains Map<String, SectionResultModel>

Can we adjust it somehow this way:
InfoResultModel  contains List<String>
TabularResultModel contains Map<String, List<String>>
SectionResultModel contains Map<String, String>
CompositeResultModel contains Map<String, InfoResultModel/TabularResultModel/SectionResultModel>
Would this be more powerful?